chainstate/chainstate.go

```diff
@@ -9,24 +9,44 @@ import (
 	"time"
 )
 
+type HexFormatFlag byte
+
+const (
+	RawTx HexFormatFlag = 1
+	Ef    HexFormatFlag = 3
```

Comment:
Check if `3` is the correct value. I would think that it should be `2`, as represented by `(1 << 1)`.